howto/anbox/develop-addon.md

```diff
@@ -0,0 +1,45 @@
+Developing and testing addons using AMS may be time-consuming. Instead, a container with `--devmode` enabled can provide a safe environment to test addons and their scripts without having to upload the addon to the AMS.
+
+This guide explains how to use a container in development mode to develop and test an addon using Anbox runtime. See [development mode](https://discourse.ubuntu.com/t/17763#dev-mode) to learn more about development mode enabled containers.
+
+## Launch a container in development mode
+
+Start a raw container with `--devmode` enabled:
+
+```
+amc launch --devmode --raw
+```
+
+The command prints out the ID of the container. Note down the container ID for next steps.
+
+Use the `amc shell <container_id>` command to open a shell inside the container. `container_id` is the container ID from the previous step.
+
+Alternatively, you can use `amc exec <container_id> <command_options>` to directly execute commands inside the container.
+
+## Develop and test an addon
```

Comment:
Isn't that the title of the page? Does it make sense to have a section with the same name?

Reply:
I have changed it but "Develop and test addons" will be the TOC heading and the post title can be "Develop and test addons using a container in development mode"